setup.py

```diff
@@ -14,7 +14,7 @@
     packages=find_packages(),
     include_package_data=True,
     url="https://github.com/NeurodataWithoutBorders/nwbinspector",
-    install_requires=["pynwb", "natsort", "click", "PyYAML", "jsonschema"],
+    install_requires=["pynwb", "natsort", "click", "PyYAML", "jsonschema", "importlib"],
```

Comment:
It looks to me like importlib is in the Python standard library so it doesn't need to be listed in requirements

Reply:
https://github.com/NeurodataWithoutBorders/nwbinspector/runs/5776230613?check_suite_focus=true

It appears as if the specific sub-module is not included in 3.7

Reply:
Yup, 'new to version 3.8': https://docs.python.org/3.10/library/importlib.metadata.html

Reply:
would this be a good use-case for `from __future__`, or perhaps should we make this value accessible via `nwbinspector.__version__`

Reply:
Personally I'm fine setting `nwbinspector.__version__`; but while reading up on the `importlib.metadata` and why it's better, I found old discussion regarding why https://peps.python.org/pep-0396/ was rejected

Reply:
huh, where's the discussion?